src/Project/ToolRunner.ts

```diff
@@ -81,21 +140,22 @@ export class ToolRunner {
   }
 
   public getRunner(name: string, tool: string, toolargs: ToolArgs, path: string, root?: boolean) {
+    this.killedByMe = false;
+
     return new Promise<string>((resolve, reject) => {
       Logger.info(this.tag, 'Running: ' + name);
-      let cmd = undefined;
       if (root) {
```

Comment:
For my record, this code could be refactor.

Reply:
I am not fully understand what this comment meant for this PR. Could you elaborate more about what I need to do?

Reply:
Sorry for my ambiguous comment. I have a plan to revise `cmd` for `root` and `not root`. That's why I wrote the comment. Please ignore it. Sorry 😃